### PR TITLE
Change dependency requirements

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

Update dependency requirements based on https://github.com/kubesphere/kubekey/issues/282.

### CODE REVIEW

File: content/en/docs/quick-start/all-in-one-on-linux.md [hunk 1]
```diff
@@ -37,9 +37,25 @@ The system requirements above and the instructions below are for the default min
 
 - The node can be accessed through `SSH`.
 - `sudo`/`curl`/`openssl` should be used.
-- `ebtables`/`socat`/`ipset`/`conntrack` should be installed in advance.
 - `docker` can be installed by yourself or by KubeKey.
```
Comment (pixiake) [inline content/en/docs/quick-start/all-in-one-on-linux.md:40]: Should we declare that docker need to be install by themselves in an offline environment.

File: content/en/docs/installing-on-linux/introduction/multioverview.md [hunk 2]
```diff
@@ -56,9 +56,19 @@ The path `/var/lib/docker` is mainly used to store the container data, and will
...
 - Time synchronization for all nodes.
 - `sudo`/`curl`/`openssl` should be used in all nodes.
-- `ebtables`/`socat`/`ipset`/`conntrack` should be installed in all nodes.
 - `docker` can be installed by yourself or by KubeKey.
 
+### Dependency Requirements
+
+KubeKey can install Kubernetes and KubeSphere together. The dependency that needs to be installed may be different based on the Kubernetes version to be installed. You can refer to the list below to see if you need to install relevant dependencies on your node in advance. 
+
+|             | Kubernetes Version ≥ 1.18 | Kubernetes Version < 1.18 |
```
Comment (Forest-L) [inline content/en/docs/installing-on-linux/introduction/multioverview.md:65]: Dependencies can be added with a title

File: content/en/docs/quick-start/all-in-one-on-linux.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/docs/quick-start/all-in-one-on-linux.md:40]: Added in a note to highlight this hint.

File: content/en/docs/installing-on-linux/introduction/multioverview.md (hunk 2 above)
Comment (Sherlock113) [reply content/en/docs/installing-on-linux/introduction/multioverview.md]: Added

**Pull request merged**